starter_code/elevator.js

```diff
@@ -1,19 +1,60 @@
 class Elevator {
-  constructor(){
-    this.floor      = 0;
-    this.MAXFLOOR   = 10;
-    this.requests   = [];
+  constructor() {
+    this.floor = 0;
+    this.MAXFLOOR = 10;
+    this.direction = "up";
+    this.requests = [];
+    this.requestedFloors = [];
```

Comment:
Tienes arrays de sobra. Solo hace falta un array de requests. En este array introduces el piso desde el que te han hecho el request.

Reply:
El enunciado dice: Add the whole person object to the requests array. The elevator will need all that information later. 
No entendía y por eso he creado otra con solo el piso.